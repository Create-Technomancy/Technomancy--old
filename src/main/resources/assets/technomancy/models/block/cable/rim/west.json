{
	"credit": "Made with Blockbench",
	"texture_size": [32, 32],
	"textures": {
		"0": "technomancy:block/cables",
		"particle": "technomancy:block/cables"
	},
	"elements": [
		{
			"name": "rim",
			"from": [0, 4, 4],
			"to": [2, 12, 12],
			"rotation": {"angle": 0, "axis": "y", "origin": [8, 8, 8]},
			"faces": {
				"north": {"uv": [9, 0, 8, 4], "texture": "#0"},
				"east": {"uv": [0, 0, 4, 4], "texture": "#0"},
				"south": {"uv": [8, 0, 9, 4], "texture": "#0"},
				"west": {"uv": [4, 0, 8, 4], "texture": "#0"},
				"up": {"uv": [8, 0, 9, 4], "texture": "#0"},
				"down": {"uv": [8, 0, 9, 4], "texture": "#0"}
			}
		}
	]
}