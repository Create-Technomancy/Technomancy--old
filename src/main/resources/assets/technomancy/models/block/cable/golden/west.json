{
	"credit": "Made with Blockbench",
	"texture_size": [32, 32],
	"textures": {
		"0": "technomancy:block/cables",
		"particle": "technomancy:block/cables"
	},
	"elements": [
		{
			"name": "west",
			"from": [0, 5, 5],
			"to": [5, 11, 11],
			"rotation": {"angle": 0, "axis": "y", "origin": [8, 8, 8]},
			"faces": {
				"north": {"uv": [3, 7.5, 6, 10], "rotation": 270, "texture": "#0"},
				"south": {"uv": [0, 7.5, 3, 10], "rotation": 270, "texture": "#0"},
				"up": {"uv": [0, 7.5, 3, 10], "rotation": 270, "texture": "#0"},
				"down": {"uv": [0, 7.5, 3, 10], "rotation": 270, "texture": "#0"}
			}
		}
	]
}